### PR TITLE
Add A+ goreportcard and fix bash syntax

### PR DESCRIPTION
**Pull request by sectioneight**
Let's be pround of our work.

### CODE REVIEW

**Comment by sectioneight** [comment]
https://github.com/jstemmer/go-junit-report/blob/0c1d4ddfcfc1239c1b0d71adadea5378cd8d405a/README.md

full preview here

/r @jstemmer @rmmh

**Comment by sectioneight** [comment]
https://github.com/jstemmer/go-junit-report/blob/0c1d4ddfcfc1239c1b0d71adadea5378cd8d405a/README.md

full preview here

/r @jstemmer @

**Comment by rmmh** [comment]
Hah, sure. :-)

I see passing/A+ now.

File: README.md [hunk 1]
```diff
@@ -3,20 +3,28 @@
...
+go get -u github.com/jstemmer/go-junit-report
+```
 
 ## Usage
 
 go-junit-report reads the `go test` verbose output from standard in and writes
 junit compatible XML to standard out.
 
-	go test -v | go-junit-report > report.xml
+```bash
```
Comment (rmmh) [inline README.md]: Highlighting "test" as a keyword feels more distracting than the subtle coloring of | and > is useful.

Comment (sectioneight) [inline README.md]: Thinking just plain triple backticks with no language hint?

Comment (rmmh) [inline README.md]: I guess it's a nice hint to people what language we're using.

Comment (rmmh) [inline README.md]: We can put the solutions for #37 in the README too.

**Comment by rmmh** [comment]
LGTM.

What's your preferred merge strategy? Do you prefer linear history (using the rebase option)?

**Comment by sectioneight** [comment]
I prefer squash (linear).

**Comment by sectioneight** [comment]
Doesn't look like I get a choice though. Can you or @jstemmer enable squash commits? Or should we just all do merge?

**Pull request merged**